sample-app/src/main/java/com/joinforage/android/example/ui/pos/data/PosPaymentRequest.kt

```diff
@@ -0,0 +1,25 @@
+package com.joinforage.android.example.ui.pos.data
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class PosPaymentRequest(
+    val amount: Double,
+    @Json(name = "funding_type") val fundingType: String,
+    @Json(name = "payment_method") val paymentMethodRef: String,
+    val description: String,
+    @Json(name = "pos_terminal") val posTerminal: PosTerminal,
+    val metadata: Map<String, String>
+)
```

Comment:
This is great because it should be easy to extend this to support all of the withdrawal-related flows coming up. For example [this one](https://linear.app/joinforage/issue/FX-748/story-cash-withdrawal-flow), which requires that we pass a  `transaction_type="withdrawal"`